src/main/resources/org/sonar/plugins/findbugs/rules-findsecbugs.xml

```diff
@@ -2200,15 +2200,15 @@ to do this correctly.
     <tag>security</tag>
   </rule>
   <rule key='DES_USAGE' priority='MAJOR'>
-    <name>Security - DES/DESede is insecure</name>
+    <name>Security - DES is insecure</name>
     <configKey>DES_USAGE</configKey>
     <description>&lt;p&gt;
-DES and DESede (3DES) are not considered strong ciphers for modern applications. Currently, NIST recommends the 
-usage of AES block ciphers instead of DES/3DES.
+DES is considered strong ciphers for modern applications. Currently, NIST recommends the
```

Comment:
This is incorrect, DES is not considered a strong cipher for modern applications. 😉

Reply:
Plz feedback to https://find-sec-bugs.github.io

Reply:
As it's a problem with the Sonar description and not the rule itself, I'm unsure how this is relevant to https://find-sec-bugs.github.io